pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
@rhcarvalho if you want to limit the width of the comments to 69 chars, doing so in PR titled with "Add build trigger information" is not a great idea. First of the changes are unrelated to the PR, second it make the PR hard to review (i'm not going to review each commit separately a comment on each commit ;-). If you want to make these changes, a separate PR is preferred, presumably fixing this in all `types.go`.